### DISCUSSION TITLE
Horizontal scaling based on microservice Memory usage

### DISCUSSION THREAD

**Discussion by elmalakomar** [mia-platform/community#324] [Ideas]
### Which area of this project could be most improved?

Console

### Description

Hi,
With a client recently we had issues with the horizontal scaling since we needed an autoscaling based on memory usage. At this moment the Console supports only CPU autoscaling.

It could be a nice feature to introduce to the console. What do you think?

**Comment by fredmaggiowski** [comment]
Hi, the HorizontalPodAutoscaler [v2 API (autoscaling/v2)](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#scaling-on-multiple-metrics) should now have stable support for memory based metrics; with the new project workflow it should be possible to create them if the cluster has the correct k8s api version; I think we can put this feature request in roadmap, can you open it?

@albertotessarotto FYI

**Reply by albertotessarotto** [reply]
Thanks for the proposal, I traced the request in our internal roadmap

**Comment by akelity** [comment]
This feature has been positively evaluated and may be considered for inclusion in the roadmap in the future, depending on priorities and overall specific needs.
